superset-frontend/src/explore/components/controls/FilterControl/AdhocFilterEditPopoverSimpleTabContent/index.tsx

```diff
@@ -346,7 +349,12 @@ const AdhocFilterEditPopoverSimpleTabContent: React.FC<Props> = props => {
           endpoint: `/superset/filter/${datasource.type}/${datasource.id}/${col}/`,
         })
           .then(({ json }) => {
-            setSuggestions(json);
+            setSuggestions(
+              json.map((suggestion: null | number | boolean | string) => ({
+                value: suggestion,
+                label: optionLabel(suggestion),
```

Comment:
Shouldn't we translate the option labels? I notice that the `optionLabel` function is always returning the values in English.

Reply:
..then the translation needs to be done where the constant is defined. And if we do that, then we should definitely remove support for using the strings `"<NULL>"` and `"<empty string>"` as proxies for `None` and `""` respectively in the backend and assume they'll always just be labels, but never values.